pkg/envoy/admin/testdata/golden.full_config.json

```diff
@@ -7,41 +7,41 @@
           "id": "default.backend-1",
```

Comment:
When compared with the source file it has only one change:
<img width="1515" alt="image" src="https://github.com/kumahq/kuma/assets/13002762/e37ac733-bd0a-4f15-9857-9866a8db3bc7">